src/tilemaker/scripts/run_tilemaker.sh

```diff
@@ -33,21 +33,67 @@ function generate_tiles_from_shapefile() {
 
     # Get bounds. ogrinfo can handle big files, but doesn't provide tons of output flexibility, so
     # parse out the answer from the blob of text it returns.
-    TL_BOUNDS=$(ogrinfo -so -al "/data/${TL_SHAPEFILE_NAME}.shp" \
+    export TL_BOUNDS=$(ogrinfo -so -al "/data/${TL_SHAPEFILE_NAME}.shp" \
         | grep Extent | sed -E 's/.*\((.*), (.*)\) - \((.*), (.*)\).*/\1 \2 \3 \4/')
 
     # Make tiles and upload them to S3.
-    # 'tl' only has totally-quiet or very-verbose (every tile) output options, so this filters
+    # 'tl' only has totally-quiet or very-verbose (every tile) output options, so we filter
     # the output to show the first line then every 1000th line thereafter.
-    /usr/bin/time -f "\nTIMING: %C\nTIMING:\t%E elapsed %Kkb mem\n" \
-    tl copy --min-zoom "${TL_MIN_ZOOM}" --max-zoom "${TL_MAX_ZOOM}" \
-        --bounds "${TL_BOUNDS}" \
-        "mapnik:///opt/pfb/tilemaker/styles/${TL_SHAPEFILE_NAME}_style.xml" \
-        "s3://${AWS_STORAGE_BUCKET_NAME}/${PFB_S3_TILES_PATH}/${TL_SHAPEFILE_NAME}/{z}/{x}/{y}.png" \
-        | awk 'NR == 1 || NR % 1000 == 0'
+    #
+    # The work is done in the two functions, which are spawned in many (relatively) bit-sized
+    # pieces with `parallel`
+
+    # Define these once to save space below. They need to be exported because they'll be used
+    # in the functions.
+    export TL_INPUT="mapnik:///opt/pfb/tilemaker/styles/${TL_SHAPEFILE_NAME}_style.xml"
+    export TL_OUTPUT="s3://${AWS_STORAGE_BUCKET_NAME}/${PFB_S3_TILES_PATH}/${TL_SHAPEFILE_NAME}/{z}/{x}/{y}.png"
```

Comment:
Ok, I picked a tiled job arbitrarily and got Memphis, which saved its 70,000 tiles over the course of 78 minutes.  So that's 15/second from one thread.  (Also, so many tiles.  17 is seeming like a good choice unless/until we definitely need 18.)  So 15 4-threaded jobs running would produce about 900 PUTs per second.  If we ended up running a re-tiling job and had 20 instances with two jobs each, that could mean 2,400/s.  Interesting.